stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -84,8 +84,8 @@ public void prepare() throws Exception {
           "fixed count must be > 0. fixedCount: " + mParameters.mFixedCount);
     }
 
-    if (!mParameters.mProfileAgent.isEmpty()) {
-      mBaseParameters.mJavaOpts.add("-javaagent:" + mParameters.mProfileAgent
+    if (!mBaseParameters.mProfileAgent.isEmpty()) {
```

Comment:
Since this is now in base parameters, this block can be moved to `Benchmark`, right?